### ISSUE TITLE
allow AWS SSO in private browser window/tab

### ISSUE THREAD

**Issue by it-ben** [Noovolari/leapp#412]
**Is your feature request related to a problem? Please describe.**
If your AWS SSO is connected to your AD, your browser is already authenticated via Windows logon. But if you won't use this account, because it is not allowed to logon to AWS, it is not possible to sign in with the right account.

**Describe the solution you'd like**
Leapp should offer the possibility to open a private tab or displaying the URL with the authentication code like https://device.sso.region.amazonaws.com/?user_code=ABCD-EFGH

**Describe alternatives you've considered**
I've tried to catch the code URL, but once this code is used because of automatic forwarding it cant used twice.

**Additional context**
The app is very cool! =)

**Comment by andreacavagna01** [comment]
Thanks for the enhancement! Have you tried in-app authentication for the Integration?

**Comment by it-ben** [comment]
> Thanks for the enhancement! Have you tried in-app authentication for the Integration?

yes I've tried in-app already. This is using my Windows credentials too.

**Comment by toddwalters** [comment]
+1 on this request default browser and in-app auth both default to using regular user credentials.  Only way for me to work around this is to open incognito/private browser window/tab.

**Comment by andreacavagna01** [comment]
Could be an idea to create a copy sign-in link in the authentication in browser? 

in that way you can use that link wherever you want to login with Leapp, anonymous browser or not
